tests/api-tests/fields/required.test.ts

```diff
@@ -53,6 +52,11 @@ for (const modulePath of testModules) {
               fields: {
                 name: text(),
                 testField: mod.typeFunction({
+                  ...(mod.nonNullableDefault ? {
+                    db: {
+                      isNullable: true
```

Comment:
These tests are written around the premise that `db.isNullable` is `true` by default